proto/src/determined/api/v1/run.proto

```diff
@@ -55,3 +51,44 @@ message SearchRunsResponse {
   // Pagination information of the full dataset.
   Pagination pagination = 2;
 }
+
+// Message for results of individual runs in a multi-run action.
+message RunActionResult {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "error", "id" ] }
+  };
+  // Optional error message.
+  string error = 1;
+  // run ID.
+  int32 id = 2;
+}
+
+// Request to move the run to a different project.
+message MoveRunsRequest {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: {
+      required: [ "source_project_id", "destination_project_id", "run_ids" ]
+    }
+  };
+
+  // The ids of the runs being moved.
+  repeated int32 run_ids = 1;
+  // The id of the current parent project.
+  int32 source_project_id = 2;
```

Comment:
source_project_id is used to cover the case where a user tries to move a run that exists but doesn't exist in the source project.

Reply:
From an API perspective I don't see the issue moving runs from different projects. 

I'm not understanding when that case would come up and the reason we need to prevent this case.

Reply:
A scenario could be: If user 1 moves a run from project A to project B and user 2 (who's table hasn't refreshed yet, and does not have permissions on project B) tries to move the run from project A to project C.

Reply:
In that case they would fail the permission check for delete / view on that run independent of any source_project_id

I'm curious why MoveExperiment doesn't need it and this does?

Reply:
Another reason why we would need the source project id is for the filter. MoveExperiments uses BulkFilter which limits how it can be used by frontend. We want to have the same filtering that is available to SearchRuns.